pkg/registry/core/service/storage/rest_test.go

```diff
@@ -1322,93 +894,66 @@ func TestAllocateLoadBalancerNodePorts(t *testing.T) {
 
 func TestServiceRegistryDelete(t *testing.T) {
 	ctx := genericapirequest.NewDefaultContext()
-	storage, registry, server := NewTestREST(t, nil, singleStackIPv4)
+	storage, server := NewTestREST(t, singleStackIPv4)
 	defer server.Terminate(t)
-	svc := &api.Service{
-		ObjectMeta: metav1.ObjectMeta{Name: "foo"},
-		Spec: api.ServiceSpec{
-			Selector:        map[string]string{"bar": "baz"},
-			SessionAffinity: api.ServiceAffinityNone,
-			Type:            api.ServiceTypeClusterIP,
-			Ports: []api.ServicePort{{
-				Port:     6502,
-				Protocol: api.ProtocolTCP,
-			}},
-		},
+	svc := svctest.MakeService("foo")
+	_, err := storage.Create(ctx, svc, rest.ValidateAllObjectFunc, &metav1.CreateOptions{})
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
 	}
-	registry.Create(ctx, svc, rest.ValidateAllObjectFunc, &metav1.CreateOptions{})
-	storage.Delete(ctx, svc.Name, rest.ValidateAllObjectFunc, &metav1.DeleteOptions{})
-	if e, a := "foo", registry.DeletedID; e != a {
-		t.Errorf("Expected %v, but got %v", e, a)
+	_, _, err = storage.Delete(ctx, svc.Name, rest.ValidateAllObjectFunc, &metav1.DeleteOptions{})
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
 	}
 }
```

Comment:
Wouldn't better if we do another `Get` here?

Reply:
As we delayer we'll use more normal tests, so I will defer this for now